app/views/decisions/_reviewer_decision_form.html.erb

```diff
@@ -1,31 +1,29 @@
 <%= form_with model: [planning_application, decision], local: true, class: "decision" do |form| %>
 <div class="govuk-form-group">
   <fieldset class="govuk-fieldset">
-    <h2 class="govuk-heading-m">Permitted development requirements</h2>
+    <h2 class="govuk-heading-m">Review the recommendation</h2>
     <p class="govuk-body">
-      The applicant has submitted the following application
-    </p>
-    <%= render "policy_consideration_list", policy_considerations: policy_evaluation.policy_considerations %>
-    <p class="govuk-body">
-      Based on the evidence given by applicant, the planning officer recommended that permitted development should be
-      <strong><%= "#{assessor_decision.status}." %></strong>
+      The recommended by the planning officer is
```

Comment:
This is correct according to the Pivotal story, but it should be either "The action recommended..." or "The recommendation" - please can you check which one is preferred.

Reply:
We agreed to replace it with "The planning officer recommends that the application is granted/ refused and added this comment:"